examples/cpp/Example_5_5_RiftTimewarp.cpp

```diff
@@ -1,128 +0,0 @@
-#include "Common.h"
```

Comment:
Because it now only differs from Example_5_4 by a single line.  When this
example was in Chapter 5, it made sense to contrast it against 5_4; but now
that it's in Chapter 6, there's no reason to keep it as complex as it is to
compare it to the previous demo.  Once I eliminated all the complexity, I
was left with RiftApp.  There wasn't anything left in this file.

On Sat, Feb 28, 2015 at 7:39 PM, Brad Davis <notifications@github.com>
wrote:

> In examples/cpp/Example_5_5_RiftTimewarp.cpp
> <https://github.com/OculusRiftInAction/OculusRiftInAction/pull/72#discussion_r25560478>
> :
>
> > @@ -1,128 +0,0 @@
> > -#include "Common.h"
>
> Why was this file removed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OculusRiftInAction/OculusRiftInAction/pull/72/files#r25560478>
> .
>